tests/rules/import-alias.test.ts

```diff
@@ -79,19 +78,13 @@ function runTests(platform: "win32" | "posix") {
                 code: `export * from '../src-app/rules/potato';`,
                 filename: "src/test.ts",
             },
-            // selects correct alias despite #src being a partial match
-            // and comes first/is shorter as an alias
-            {
-                code: `export * from '../src-test/rules/potato';`,
-                filename: "src/test.ts",
-            },
             // does not affect source-less exports
             {
                 code: `export default TestFn = () => {}`,
                 filename: "src/test.ts",
             },
 
-            // relative path overridden for root and exports from sibling module
+            // relative path overridde for root, exporting a sibling module
```

Comment:
Looks like there's an edit that caused a recurring typo `overridden` -> `overridde`. Might to find replace `'s/overridde/override/g'` (with fd/sed/your choice).

Reply:
i replaced back to relative path overridden for root,. Sorry i'm not english native, but i guess relative path override for root, was clearer